roles/cnf_cert/tasks/pre-run.yml

```diff
@@ -111,6 +111,7 @@
       ansible.builtin.shell: |
         set -x
         podman build -t {{ tnf_image }} \
+        --force-rm \
```

Comment:
are you sure this does what you think it does? this flag defaults to true

Reply:
```
   --force-rm
       Always remove intermediate containers after a build, even if the  build  fails  (de‐
       fault true).
```

Reply:
well, I didn't realize about the default. Will close this. Thanks